File: problem-2/solution.js
var sum = 0;
var i = 0;

for (var j = 1; j <= 4000000;) {
	j = j + i;
	i = j - i;

	if ( j % 2 === 0) {
		sum += j;
	}
}

console.log(sum);

File: problem-2/solution.rb
def evenFibonacci
	sum = 0
	term1 = 1
	term2 = 2
	fibonacciArray = [1, 2]
	while term2 <= 4000000
		term2 = term1 + term2
		term1 = term2 - term1
		fibonacciArray = fibonacciArray.push term2
	end
	fibonacciArray.each do |x|
		if x % 2 == 0
			sum = sum + x
		end
	end
	puts sum
end

puts evenFibonacci